_old/planning.adoc

```diff
@@ -4,7 +4,7 @@ Quick Start deployments require a moderate level of familiarity with AWS service
 
 === AWS account
 
-If you don’t already have an AWS account, create one at https://aws.amazon.com/[https://aws.amazon.com^] by following the on-screen instructions. Part of the sign-up process involves receiving a phone call and entering a PIN using the phone keypad.
+If you don’t have an AWS account, create one at https://aws.amazon.com/[https://aws.amazon.com^] by following the on-screen instructions. Part of the sign-up process involves receiving a phone call and entering a PIN.
```

Comment:
I hadn't thought of that. Good point if the CSS can make the apostrophes consistent in the built doc. CSS wouldn't address the discrepancy in searching within the .adoc files, though. It would be easy to fix all those legacy instances in the boilerplate: a one-time combthrough to replace curly apostrophes and quotation marks.